### ISSUE TITLE
[Bug]: createCloudflareKVSessionStorage isn't setting `expires` correctly

### ISSUE THREAD

**Issue by kentcdodds** [remix-run/remix#839]
### Which Remix packages are impacted?

- [ ] `remix` (Remix core)
- [ ] `create-remix`
- [ ] `@remix-run/architect`
- [X] `@remix-run/cloudflare-workers`
- [ ] `@remix-run/dev`
- [ ] `@remix-run/express`
- [ ] `@remix-run/netlify`
- [ ] `@remix-run/node`
- [ ] `@remix-run/react`
- [ ] `@remix-run/serve`
- [ ] `@remix-run/server-runtime`
- [ ] `@remix-run/vercel`

### What version of Remix are you using?

Every

### Steps to Reproduce

```ts
export const kvSessionStorage = createCloudflareKVSessionStorage({
  kv: KV_SESSION,
  cookie: {
    name: "__session",
    secure: NODE_ENV === "production",
    secrets: [SESSION_SECRET],
    sameSite: "lax",
    path: "/",
    maxAge: 60 * 60 * 24 * 7, // sessions expire after a week <-- I'm setting the maxAge
    httpOnly: true,
  },
});
```

### Expected Behavior

I expect the `maxAge` to be able to control the `expires` option in KV.

### Actual Behavior

I get the following error:

```
Error: KV PUT failed: 400 Invalid expiration of 632. Please specify integer greater than the current number of seconds since the UNIX epoch.
```

Right now, [the code](https://github.com/remix-run/remix/blob/a317b79132c8854f11e6ce334803e90fafbfc2e7/packages/remix-cloudflare-workers/sessions/cloudflareKVSessionStorage.ts#L49) sets the expires value based on what it gets in the `expires` parameter. So maybe the problem actually lies somewhere else. Not sure.

**Comment by GregBrimble** [comment]
Two bugs in one! Currently, one [L49](https://github.com/remix-run/remix/blob/57949fe0d73f79811b207fb435657da93f5306d6/packages/remix-cloudflare-workers/sessions/cloudflareKVSessionStorage.ts#L49) and [L66](https://github.com/remix-run/remix/blob/57949fe0d73f79811b207fb435657da93f5306d6/packages/remix-cloudflare-workers/sessions/cloudflareKVSessionStorage.ts#L66), data is being written to KV like this:

```ts
await kv.put(id, JSON.stringify(data), {
  expiration: expires?.getUTCMilliseconds()
});
```

1. [It should be seconds and not milliseconds](https://developers.cloudflare.com/workers/runtime-apis/kv#expiring-keys).
2. And it `getUTCMilliseconds()` just [returns the milliseconds portion of time value](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCMilliseconds), not the time in milliseconds.

It should instead be:

```ts
await kv.put(id, JSON.stringify(data), {
  expiration: expires ? Math.round(expires.getTime() / 1000) : undefined
});
```

I'll put up a quick PR.

**Comment by GregBrimble** [comment]
Dupe of #414.

**Comment by chaance** [comment]
Should be fixed in 1.1.0
